rocprofiler/PKGBUILD

```diff
@@ -11,19 +11,21 @@ makedepends=('cmake' 'python-argparse' 'python-cppheaderparser')
 options=(!staticlibs strip)
 _git='https://github.com/ROCm-Developer-Tools/rocprofiler'
 source=("$pkgname-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz"
-        'add_string_header.patch')
-sha256sums=('c42548dd467b7138be94ad68c715254eb56a9d3b670ccf993c43cd4d43659937'
-            '40264623a8431f63662137963852e1e6c025e1960212cdf4b08ce00a370b7cc4')
+        'add_string_header.patch::https://patch-diff.githubusercontent.com/raw/ROCmSoftwarePlatform/hsa-class/pull/2.patch')
```

Comment:
nit: indentation

Reply:
You mean `add_string_header.patch`? I don't see a problem in that line.